lib/fortnox/api/repositories/base.rb

```diff
@@ -111,7 +111,7 @@ def expired?(token)
           decoded_token = JWT.decode token, nil, false
           decoded_token[0]['exp'] < (Time.now.to_i - TIME_MARGIN_FOR_ACCESS_TOKEN_RENEWAL)
         rescue JWT::DecodeError
-          raise Exception, "Could not decode access token for token store \"#{@token_store_name}\""
+          raise Fortnox::API::Exception, "Could not decode access token for token store \"#{@token_store_name}\""
```

Comment:
Why do you need this if you have:
```
Lint/RaiseException:
  AllowedImplicitNamespaces: ['Fortnox::API']
```

Reply:
We don't. Good catch 👍

Reply:
Unfortunately, I can't get this to work. I have this config:
```
Lint/RaiseException:
  AllowedImplicitNamespaces: ['Fortnox::API']
```
but I still get this error:
```
lib/fortnox/api/repositories/base.rb:114:17: W: [Correctable] Lint/RaiseException: Use StandardError over Exception.
          raise Exception, "Could not decode access token for token store \"#{@token_store_name}\""
                ^^^^^^^^^
```

Maybe Rubocop is not smart enough. In the [documentation](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/RaiseException) they have an example of `Gem::Exception`, but that exception is directly in a module named `Gem`. Here, we have a class named `Fortnox::API::Repository::Base` that uses the `Fortnox::API::Exception` exception. Don't know how to fix it. I will use the fully qualified name for now...